### ISSUE TITLE
The context item for axis step ./@class is absent in "output-message.xsl"

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#2526]
I tried using the latest Saxon 9.7.0.11 to produce WebHelp output from DITA content, in one of our automated test cases this breaks with an error like:

>[xslt] D:\projects\eXml\frameworks\dita\DITA-OT2.x\xsl\common\output-message.xsl:60:6: Fatal Error! The context item for axis step ./@class is absent
>[xslt] Failed to process D:\projects\eXml\test\dita\EXM-33281\temp\webhelp\hierarchy.ditamap

Looking in the "output-message.xsl" at the template named **output-message** at some point it selects something like:

```xml
<xsl:variable name="localclass" select="@class"/>
```
but the template is sometimes called as a fallback from functions so why does it not use the $ctx node to run the xpath like:
```xml
<xsl:variable name="localclass" select="$ctx/@class"/>
```

Also the `localclass` variable does not appear to be used at all.

**Comment by lionelmoi** [comment]
I have the same issue and I cannot see where this variable is used.

**Comment by jelovirt** [comment]
Fixed by  #2535
